src/urdf.jl

```diff
@@ -37,43 +37,40 @@ function parse_geometries(xml_geometry::XMLElement, package_path, file_path="")
     end
     for xml_mesh in get_elements_by_tagname(xml_geometry, "mesh")
         filename = attribute(xml_mesh, "filename")
-        dae_pattern = r".dae$"
-        replaced_extension_with_obj = false
-        if occursin(dae_pattern, filename)
-            filename = replace(filename, dae_pattern => ".obj")
-            replaced_extension_with_obj = true
-        end
         package_pattern = r"^package://"
-
         if occursin(package_pattern, filename)
             found_mesh = false
             for package_directory in package_path
-                filename_in_package = joinpath(package_directory, replace(filename, package_pattern => ""))
-                if ispath(filename_in_package)
-                    mesh = load(filename_in_package, GLUVMesh)
-                    push!(geometries, mesh)
-                    found_mesh = true
-                    break
+                basename, ext = splitext(joinpath(package_directory, replace(filename, package_pattern => "")))
+                for ext_to_try in [ext, ".obj"] # TODO: remove this once other packages are updated
```

Comment:
This is because the AtlasRobot and ValkyrieRobot URDFs currently lie and say that the file extension is .dae, while there are only .obj files.

Reply:
Heh, yeah. Agreed that we should just fix those URDFs and then remove this.